prism-agent/service/server/src/main/scala/io/iohk/atala/iam/authentication/Authenticator.scala

```diff
@@ -45,14 +48,26 @@ trait Authenticator[E <: BaseEntity] {
 }
 
 trait Authorizer[E <: BaseEntity] {
-  def authorize(entity: E): IO[AuthenticationError, WalletAccessContext]
+  protected def authorizeWalletAccessImpl(entity: E): IO[AuthenticationError, WalletAccessContext]
```

Comment:
*Impl looks like convention for the concrete implementation of the trait
I would think to rename `authorizeWalletAccessImpl` and `authorizeWalletAccess` methods in another way.
One idea is to rename `authorizeWalletAccessImpl` to `authorizeWalletAccess` and `authorizeWalletAccess` to `authorizationLogic`, or something like that.

Reply:
Thanks for the suggestion, I was thinking hard about the name :smile: . This looks like a better fit.